src/main/kotlin/network/warzone/mars/player/achievements/gui/AchievementMenu.kt

```diff
@@ -0,0 +1,188 @@
+package network.warzone.mars.player.achievements.gui
+
+import network.warzone.api.database.models.Achievement
+import network.warzone.mars.player.achievements.AchievementDebugger
+import network.warzone.mars.player.achievements.AchievementManager.filterAchievementsWithParent
+import network.warzone.mars.player.achievements.AchievementManager.getAchievementsForCategory
+import network.warzone.mars.player.achievements.AchievementManager.getParentsFromAchievements
+import network.warzone.mars.player.achievements.models.AchievementParent
+import network.warzone.mars.player.feature.PlayerFeature
+import network.warzone.mars.player.models.PlayerProfile
+import network.warzone.mars.utils.menu.*
+import org.bukkit.Material
+import org.bukkit.enchantments.Enchantment
+import org.bukkit.entity.Player
+import org.bukkit.event.Listener
+import org.bukkit.event.inventory.InventoryClickEvent
+import org.bukkit.inventory.ItemFlag
+import kotlin.math.min
+
+class AchievementMenu(private val player: Player) : Listener, AchievementDebugger {
+    private val profile: PlayerProfile = PlayerFeature.getCached(player.name)!!
+
+    fun openMainMenu(): GUI {
+        return gui("§6§lAchievements", 1) {
```

Comment:
Please use the Bukkit ChatColor enum rather than the section symbol (throughout the rest of the class)

Example usage [here](https://github.com/Warzone/mars/blob/737f8ad9794646ec8a9777cd2d476eccc5ceb8b0/src/main/kotlin/network/warzone/mars/punishment/commands/PunishCommands.kt#L243C20-L243C20)

Reply:
Well regarding this, I currently have the "wrap" function refactored to make use of the section symbol in coloring menu descriptions. Would I be able to just replace those section symbols with the ChatColor enum, or do you think it would require additional refactoring to get that to work with the wrap function?

Reply:
I believe it should still work with the ChatColor enum, but it's worth testing to make sure

Reply:
You can always use `ChatColor.COLOR_CHAR` for your wrap function as well, which is defined to be the section sign